c-dependencies/js-compute-runtime/xqd.wit

```diff
@@ -486,4 +486,8 @@ default world xqd-world {
 
     purge-surrogate-key: func(surrogate-key: string, soft-purge: bool) -> result<purge-result, error>
   }
+
+  export fastly-runtime: interface {
+    serve-sync: func() -> ()
+  }
```

Comment:
Would it be possible to have this in a different file which imports the xqd world?
The reason I ask is because I think it would be good to keep the xqd definition in a different file to help make it clear that the xqd definitions are not ours (also we should probably name it compute-at-edge instead of xqd 🙈 )

Reply:
Agreed, we could even explicitly name it exerpimental-compute-at-edge or something to indicate it's undergoing change.

Reply:
Agreed, we could even explicitly name it exerpimental-compute-at-edge or something to indicate it's undergoing change.